### ISSUE TITLE
Crash at start with out of memory

### ISSUE THREAD

**Issue by ac2cz** [ac2cz/FoxTelem#596]
Alan says: 
I updated my various versions for FoxTelem.  The updates went fine, except for the one version which uses the default FoxTelem.exe to start.  In this case, it gets the “Loading Data” box and gets to 75%, then this error message:
![image](https://user-images.githubusercontent.com/14824793/67405523-e3540a00-f582-11e9-815a-b89f525a6d80.png)

I can OK the box which closes, but it remains at 75% and I need to use the Task Manager to close it.  This is the only one which is set up for AF feed.

**Comment by ac2cz** [comment]
Note that clicking OK really should exit the application so the user does not need to use Task Manager.

**Comment by ac2cz** [comment]
I will regress to the 1.07 FoxTelem.exe.  This was designed to manage the memory more tightly.  It needs more thought and testing.  Moving to 1.09

**Comment by ac2cz** [comment]
Downgraded to High because this is not an issue with the standard launcher.  It is an opportunity to default to a lower memory footprint if I can get this right

**Comment by ac2cz** [comment]
Just as a reminder, this issue was caused when the Java EXE launcher was changed to default to a lower default memory usage.  When run with the default params it seems to report about 100MB of memory in Task Manager.  I had set the launcher to default to 32MB with a max of 128M.  The max should have been much higher.  e.g. 512MB or even 1024MB.  

But with those settings we still use ~100MB.  With no data at all it uses 90MB.  So a change like this seems too risky for perhaps 10MB of saved memory at launch. So this is rejected..
